JPS_VIRTUALSENSOR/DispersionInteractor/src/main/java/com/cmclinnovations/dispersion/InitialiseSimulation.java

```diff
@@ -57,6 +57,10 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
         String ewkt = req.getParameter("ewkt");
         int nx = Integer.parseInt(req.getParameter("nx"));
         int ny = Integer.parseInt(req.getParameter("ny"));
+        String citiesNamespace = null;
+        if (req.getParameter("citiesnamespace") != null){
+            citiesNamespace = req.getParameter("citiesnamespace");
+        }
```

Comment:
```suggestion
        String citiesNamespace = req.getParameter("citiesnamespace");
```

Reply:
I believe this should give the same result

Reply:
Done.